src/app/index.ts

```diff
@@ -0,0 +1,9 @@
+import Express from 'express';
+
+const PORT= 8080;
```

Comment:
We could move the port to a configuration file (.env file)

Reply:
of course.